tools/run_tests/report_utils.py

```diff
@@ -29,6 +29,8 @@
 
 """Generate XML and HTML test reports."""
 
+from mako.runtime import Context
```

Comment:
Once you move the mako import inside the render_html_report function, run_tests.py will no longer fail.